.flowconfig

```diff
@@ -14,4 +14,3 @@ esproposal.class_static_fields=enable
 unsafe.enable_getters_and_setters=true
 module.name_mapper='.*\(.css\)' -> 'CSSModule'
 module.system=haste
-strip_root=true
```

Comment:
N.B. This is no longer a valid option in .flowconfig, although it continues to be supported as a command-line argument if you need it. It doesn't really change the output much.